x-pack/test_serverless/functional/test_suites/common/discover/group1/_discover_histogram.ts

```diff
@@ -75,7 +74,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         const actualCount = await elasticChart.getVisualizationRenderingCount();
         const expectedCount = prevRenderingCount + renderingCountInc;
         log.debug(`renderings before brushing - actual: ${actualCount} expected: ${expectedCount}`);
-        return actualCount === expectedCount;
+        return actualCount <= expectedCount;
```

Comment:
I have a question here the skipped test is failing because 

` actual: 2 expected: 1`

With this change it will pass when the actual count is 0,1, ... but actually when there's an extra rendering, that we try to catch, it is not caught. My question, is the behavior als flaky when trying it out locally without any change?

Reply:
Hi @kertal!

Currently it fails with `renderings before brushing - actual: 3 expected: 4`. 

As @davismcphee  mentioned in https://github.com/elastic/kibana/pull/178592/files we don't know why it can be 4 sometimes on serverless. Anything below 4, we can consider as a good result. Same is on line 93 https://github.com/elastic/kibana/pull/179974/files/a08221d61210bb78ffde8aa98e3d6a6a831636f3#diff-fa544caff5a4583101f0926db5dd31ce3c10983d491a97e4484126c709f715c6R93

Reply:
thx @jughosta , LGTM , we could consider to skip such tests that need very specific serverless adaptation, when they are not critical like in this case